### PR TITLE
system tests: new system-service bats file

### PR DESCRIPTION
**Pull request by edsantiago**
Followup to #14957, which added a new test that doesn't
actually belong in the 250-systemd.bats file. It was
copy-pasted from another test that doesn't belong there.

Move both tests to a new .bats file, because (1) they
need a custom cleanup, and (2) one of the tests should
very definitely run under podman-remote, and the 250
bats file has a global skip_if_remote().

Signed-off-by: Ed Santiago <santiago@redhat.com>
```release-note
None
```

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM

File: test/system/250-systemd.bats [hunk 1]
```diff
@@ -296,8 +296,6 @@ LISTEN_FDNAMES=listen_fdnames" | sort)
 }
 
 @test "podman-kube@.service template" {
-    skip_if_remote "systemd units do not work with remote clients"
-
```
Comment (Luap99) [inline test/system/250-systemd.bats:300]: Why is this removed? There is no point in testing this with podman-remote since the units have to use local podman.

Comment (edsantiago) [reply test/system/250-systemd.bats:300]: Global skip (in `setup()`). *EVERYTHING* in this file is skipped on remote. Having an explicit `skip` here serves only to confuse a reader.
